tests/unit/cmdline_interface/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@
 FILE(GLOB cmd_interface_test_src "*.cpp")
 
 add_executable(cmd_interface_test_bin ${cmd_interface_test_src})
-target_link_libraries(cmd_interface_test_bin ${MPI_C_LIBRARIES} ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} coreneuron ${reportinglib_LIBRARY})
+target_link_libraries(cmd_interface_test_bin ${MPI_CXX_LIBRARIES} ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} coreneuron ${reportinglib_LIBRARY})
```

Comment:
the MPI_CXX_LIBRARIES part is covered by #102 
nice, boost test is getting introduced!